ignite/contrib/handlers/param_scheduler.py

```diff
@@ -586,11 +612,10 @@ def simulate_values(cls, num_events, schedulers, durations, param_names=None, **
         # not perturb original scheduler.
         with tempfile.TemporaryDirectory() as tmpdirname:
             cache_filepath = Path(tmpdirname) / "ignite_lr_scheduler_cache.pt"
-            objs = {"lr_scheduler_{}".format(i): s.state_dict() for i, s in enumerate(schedulers)}
-            # all schedulers should be related to the same optimizer
-            objs["optimizer"] = schedulers[0].optimizer.state_dict()
+            scheduler_objs = {"lr_scheduler_{}".format(i): s.state_dict() for i, s in enumerate(schedulers)}
```

Comment:
Why can't we keep previous code here ?

Reply:
because `ParamGroupScheduler` could be in the list and as you mentionned above, it's a list of optimizers.